.changeset/slow-cameras-brush.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/select-field': patch
```

Comment:
```suggestion
'@commercetools-uikit/select-field': patch
'@commercetools-uikit/select-input': patch
```

Reply:
Added another changeset file for `select-input` so we have different messages per package: 6ad608a0c3884dec23057a35bda03005c368b6c9

Thanks~